examples/do_MLICA.py

```diff
@@ -0,0 +1,134 @@
+"""
+Compute ICA object based on filtered and downsampled data.
+Identify ECG and EOG artifacts using MLICA and compare
+results to correlation & ctps analysis.
```

Comment:
We can also include a reference for MLICA method

Ahmad Hasasneh, Nikolas Kampel, Praveen Sripad, N. Jon Shah, and Jürgen Dammers, “Deep Learning Approach for Automatic Classification of Ocular and Cardiac Artifacts in MEG Data,” Journal of Engineering, vol. 2018, Article ID 1350692, 10 pages, 2018. https://doi.org/10.1155/2018/1350692

Reply:
Will do.